{
  IgniteCheckedException me=prepareMultiException();
  try {
    throw new RuntimeException("Test.",me);
  }
 catch (  RuntimeException e) {
    assertNull(X.cause(e,IOException.class));
    assertNotNull(X.cause(e,IgniteCheckedException.class));
    assertTrue(X.cause(e,IgniteCheckedException.class) instanceof IgniteCheckedException);
    assertNotNull(X.cause(e,IllegalArgumentException.class));
    assertTrue(X.cause(e,IllegalArgumentException.class) instanceof IllegalArgumentException);
  }
}
